pkg/cmd/openshift-kube-apiserver/kubeadmission/register.go

```diff
@@ -43,8 +44,7 @@ var (
 	SkipRunLevelOnePlugins = sets.NewString(
 		"authorization.openshift.io/RestrictSubjectBindings",
 		"autoscaling.openshift.io/ClusterResourceOverride",
-		"autoscaling.openshift.io/RunOnceDuration",
-		"image.openshift.io/ImagePolicy",
+		imagepolicyapiv1.PluginName,
```

Comment:
add a comment with the actual string we'll be seeing as we backtrace from configuration through here.

`imagepolicyapiv1.PluginName, // "image.openshift.io/ImagePolicy"`

Reply:
will add